packages/amplication-client/src/Entity/EntityFieldLinkList.tsx

```diff
@@ -57,11 +57,11 @@ export const EntityFieldLinkList = React.memo(({ entityId }: Props) => {
             </InnerTabLink>
           </div>
         ))}
+        {data?.entity && (
+          <NewEntityField onFieldAdd={handleFieldAdd} entity={data?.entity} />
+        )}
```

Comment:
@EugeneTseitlin according to issue, the field should noe be located above the list. From what I understand you moved the field to a new location below the list.

In case I'm missing something, can you please add a screenshot or a video of the new layout after your changes?